screwdriver.yaml

```diff
@@ -0,0 +1,12 @@
+shared:
+    environment:
+        DOCKER_REPO: screwdrivercd/buildkit-rootless
+
+jobs:
+    # Build and publish Docker image
+    docker-publish:
+        requires: [~pr, publish]
```

Comment:
It should not be specified publish job in requires because the publish job does not exist now.